### PR TITLE
now top header background-image was added and

### PR DESCRIPTION
**Pull request by piotrbelling**
logo size, font of links and measurements are changed

### CODE REVIEW

File: src/css/main.css [hunk 1]
```diff
@@ -39,101 +39,144 @@ a:hover {
...
+li a {
+    margin-left: 80px;
+    font-size: 30px;
     text-decoration: none;
     color: white;
-    padding: 0 30px 0 30px;
+    padding: 0 40px 0 40px;
 }
 
 li a:hover{
```
Comment (marszychowski) [inline src/css/main.css]: not specific enough

File: src/css/main.css [hunk 2]
```diff
@@ -39,101 +39,144 @@ a:hover {
...
-}
 
+    text-decoration: underline;
 
+}
 .navigation-list {
 
     text-underline: none;
     overflow: hidden;
+    margin: 0 !important;
```
Comment (marszychowski) [inline src/css/main.css]: try without !important

File: src/css/main.css [hunk 3]
```diff
@@ -39,101 +39,144 @@ a:hover {
 
 
 nav{
-    background-color: #b41558 ;
+    background-color: #b41558  ;
```
Comment (robert-jemiolo) [inline src/css/main.css]: a po cóż te białe znaki?
Przypominam CTRL+ALT+L - automatyczne formatowanie :)

File: src/css/main.css [hunk 4]
```diff
@@ -39,101 +39,144 @@ a:hover {
...
 }
 
 #search-input{
     float: left;
     border-radius: 5px;
     border-style: solid;
-    border-color: #005580 ;
-    width: 200px;
-    height: 24px;
+    border-color: #005580  ;
```
Comment (robert-jemiolo) [inline src/css/main.css]: jw.

**Pull request closed without merging**